--vcpkg.j_son

```diff
@@ -43,5 +43,5 @@
     "aubio",
     "fmt"
   ],
-  "builtin-baseline": "14e7bb4ae24616ec54ff6b2f6ef4e8659434ea44"
+  "builtin-baseline": "acc3bcf76b84ae5041c86ab55fe138ae7b8255c7"

```

Comment:
Yes, sorry. I meant to delete this file, actually because AFAIK, the file solely existing activates manifest mode.

I don't like manifest mode because it makes caching a lot harder and/or hacky. The point of manifest mode is that it keeps a VCPKG tree for each project/port using it. What `lukka/run-vcpkg` is doing is hackily using manifest mode, together with an undocumented option to override the install path.

Frankly. I think it'd be much cleaner to update the README/wiki or even make a batch file to fetch the necessary dependencies for the user (could probably be largely copied from the CI itself)

vcpkg-action installs is globally in the same way Linux and macOS package managers work. By default, it uses latest master; I forked it and changed it because at the time I did this, master was broken. My version defaults to the latest stable release (i.e. it's a bit outdated now but would be automatically updated when a new version is released). We could force it to the current master, though.

Reply:
To be more specific, from the README of vcpkg-action: 
```
I created a simple GitHub action designed to accurately cache packages by using a "dry-run" build to generate an accurate list of packages and package versions. The dry-run is hashed as part of the cache restore key, so the cache will always accurately reflect the packages that need to be built.
```

Reply:
> because AFAIK, the file solely existing activates manifest mode.

You're correct. If it exists it will be running in manifest mode.

> Frankly. I think it'd be much cleaner to update the README/wiki or even make a batch file to fetch the necessary dependencies for the user (could probably be largely copied from the CI itself)

We also have this for linux/osx so yes this can be a solution 👍 

Personally i'm more fond of dealing with vcpkg.exe directly instead of through a github action like mentioned below
However if i'm the only one advocating for that let's use the GH-action

So basicly the todo's here are:

- [ ] 1. Change wiki to add a vcpkg.exe command to install the specified libraries from the correct vcpkg version
- [ ] 2. Remove the manifest file